docs/api_docs.rst

```diff
@@ -53,7 +53,8 @@ We provide two utility functions as baseline implementation to perform this step
 
 * ``xaitk_saliency.utils.masking.occlude_image_batch`` - performs the transformation as a batch operation
 
-* ``xaitk_saliency.utils.masking.occlude_image_streaming`` - performs the transformation in a streaming method with optional parallelization in that streaming
+* ``xaitk_saliency.utils.masking.occlude_image_streaming`` - performs the transformation in a streaming method with
+optional parallelization in that streaming
```

Comment:
The second line of text needs to align with the bullet indentation to be considered part of the same block.
```suggestion
* ``xaitk_saliency.utils.masking.occlude_image_streaming`` - performs the transformation in a streaming method with
  optional parallelization in that streaming
```